src/main/groovy/edu/oregonstate/mist/osuevents/resources/EventsResource.groovy

```diff
@@ -66,14 +74,116 @@ class EventsResource extends Resource {
  * GET all events
  */
     @GET
-    @Produces(MediaType.APPLICATION_JSON)
+    @Produces([MediaType.APPLICATION_JSON, "text/csv"])
+    //TODO Add CSV MediaType to response object
     public Response getEvents(@Auth AuthenticatedUser _,
                               @QueryParam('format') String format) {
-        def events = eventsDAO.getEvents()
-//@TODO add methods for returning csv and ics formats
-//        else if (format == "csv") {
-//
-//        } else if (format == "ics") {
+        List<ResourceObject> events = eventsDAO.getEvents()
+        def resultObject
+
+        if (!format) {
+            events.each {
+                resultObject.data += new ResourceObject(
```

Comment:
resultObject wasn't initiated as a `new ResultObject` so when a I make a request for /events, it's throwing a null pointer exception error.

Also, the way you have this setup, it's not adding any instances to each resource object. Why did you add logic for this? Can we use the getResultObject method instead? Here's what I'm thinking:
```
if (!format) {
    def events = eventsDAO.getEvents()
    return Response.ok(getResultObject(events)).type(MediaType.APPLICATION_JSON).build()
} else if (format == "csv" {
    //the rest of the stuff